tests/Http/SignupTest.php

```diff
@@ -561,59 +561,20 @@ public function testSignupShowWithIncludedPostsAndParamsAsOwner()
      * GET /api/v3/signups?include=user
      * @return void
      */
-    public function testSignupIndexWithIncludedUserAsAdmin()
+    public function testSignupIndexWithIncludedUser()
     {
         $post = factory(Post::class)->create();
         $signup = $post->signup;
 
         // Test with admin that entire user is returned.
         $response = $this->withAdminAccessToken()->getJson('api/v3/signups?include=user');
         $response->assertStatus(200);
-        $decodedResponse = $response->decodeResponseJson();
-
-        $this->assertEquals(false, empty($decodedResponse['data'][0]['user']['data']['birthdate']));
-    }
-
-    /**
-     * Test for signup index with included user info. as owner.
-     * Only admins/owners should be able to see all user info.
-     *
-     * GET /api/v3/signups?include=user
-     * @return void
-     */
-    public function testSignupIndexWithIncludedUserAsOwner()
-    {
-        $post = factory(Post::class)->create();
-        $signup = $post->signup;
 
-        // Test with admin that entire user is returned.
-        $response = $this->withAccessToken($signup->northstar_id)->getJson('api/v3/signups?include=user');
-        $response->assertStatus(200);
         $decodedResponse = $response->decodeResponseJson();
 
         $this->assertEquals(false, empty($decodedResponse['data'][0]['user']['data']['birthdate']));
     }
 
-    /**
-     * Test for signup index with included user info. as non-admin/non-owner.
-     * Only admins/owners should be able to see all user info.
-     *
-     * GET /api/v3/signups?include=user
-     * @return void
-     */
-    public function testSignupIndexWithIncludedUserAsNonAdminNonOwner()
-    {
-        $post = factory(Post::class)->create();
-        $signup = $post->signup;
-
-        // Test with annoymous user that only a user's first name is returned.
-        $response = $this->getJson('api/v3/signups?include=user');
-        $response->assertStatus(200);
-        $decodedResponse = $response->decodeResponseJson();
-
-        $this->assertEquals(true, empty($decodedResponse['data'][0]['user']['data']['birthdate']));
```

Comment:
Note: These tests are no longer necessary since Rogue itself doesn't have any authorization logic for users. (It won't ever get anything from Northstar that the user making the request isn't allowed to see).

Reply:
So this test is essentially redundant to a similar test in Northstar?